Gems/PhysX/Code/Source/EditorPrismaticJointComponent.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzFramework/Entity/EntityDebugDisplayBus.h>
+#include <AzToolsFramework/API/ComponentEntitySelectionBus.h>
+
+#include <PhysX/EditorJointBus.h>
+#include <Editor/EditorJointConfiguration.h>
+#include <Source/EditorJointComponent.h>
+
+namespace PhysX
+{
+    class EditorPrismaticJointComponent
```

Comment:
suggestion: API comment here might be useful

Reply:
Done